### DISCUSSION TITLE
Automatical responsive feature

### DISCUSSION THREAD

**Discussion by Seef0x** [WilliamVenner/vscode-glua-enhanced#27] [Ideas]
When a right click in a number (in drawing functions for example), its automatically converted into adaptable position/size to all screens

**Comment by WilliamVenner** [comment]
I like this idea, but it would just make your code really messy. Just doing something like this is much better:

```lua
local function ScaleW(w)
    return (w / 1920) * ScrW()
end
local function ScaleH(h)
    return (h / 1080) * ScrH()
end

ScaleW(800)
ScaleH(600)
...
```

Rather than having `(w / 1920) * ScrW()` and `(h / 1080) * ScrH()` everywhere.
